source/index.html

```diff
@@ -39,113 +39,115 @@
-    </dl>
-
-    <div class="map">
-      <img src="" width="" height="" alt="">
-    </div>
-
-    <a class="button" href="">Напишите нам</a>
-  </section>
+  <main class="page-main">
+    <section class="promo">
+      <h1 class="title promo__title">Милые штучки ручной работы</h1>
+
+      <a class="promo__link" href=""></a>
+      <a class="promo__link" href=""></a>
+
+    </section>
+    <section class="product-week">
+      <h2 class="product-week__title">Вязаные корзинки</h2>
+      <p class="product-week__label">Товар недели</p>
+
+      <p class="product-week__desc">
+        Экологически чистая пряжа,
+        ручная работа. Доступны в шести расцветках, и трех размерах.
+      </p>
+
+      <table class="product-week-feature">
+        <tr>
+          <td class="product-week-feature__item">
+            Цвет:
+          </td>
+          <td class="product-week-feature__item">
+            "Тиффани"/серый
+          </td>
+        </tr>
+        <tr>
+          <td class="product-week-feature__item">
+            Диаметр:
+          </td>
+          <td class="product-week-feature__item">
+            25 см
+          </td>
+        </tr>
+        <tr>
+          <td class="product-week-feature__item">
+            Высота:
+          </td>
+          <td class="product-week-feature__item">
+            15 см
+          </td>
+        </tr>
+      </table>
+
+      <img class="product-week__img">
+
+      <p class="product-week__price"></p>
+      <a class="button" href="">Заказать</a>
+
+    </section>
+    <section class="about-us">
+      <h2 class="title">Коротко о нас:</h2>
+
+      <ul class="about-us-list">
+        <li class="about-us-list__item">
+          Экологически чистые материалы
+        </li>
+        <li class="about-us-list__item">
+          Скандинавский стиль по российской цене
+        </li>
+        <li class="about-us-list__item">
+          Увеличивает лайкина фотографиях
+        </li>
+        <li class="about-us-list__item">
+          Связано вручную с любовью и умилением
+        </li>
+        <li class="about-us-list__item">
+          Поддержка отечественного производителя
+        </li>
+        <li class="about-us-list__item">
+          Поставляется в подарочной упаковке
+        </li>
+      </ul>
+
+    </section>
+    <section class="reviews">
+      <h2 class="title">Отзывы</h2>
+
+      <p class="reviews__text">
+        Я молодая мама, но всё равно фотографии не набирали и близко такого количества лайков, как
+        у популярных инстамамочек.
+        В отчаянии, я накупила аксессуаров и игрушек в Мишке,
+        и мои фотографии сразу стали более стильными, а также набирают больше лайков!
+      </p>
+
+      <p class="reviews__autor">Анастасия Красильникова <span>@misssssiskras</span></p>
+
+      <button class="reviews__button reviews__button-left">Влево</button>
+      <button class="reviews__button reviews__button-right">Вправо</button>
```

Comment:
```suggestion
      <button class="reviews__button reviews__button--left">Влево</button>
      <button class="reviews__button reviews__button--right">Вправо</button>
```
По хорошему это классы модификаторов, а не элементов. Модификатор описывает различие. Например у нас есть 2 одинаковых блока и различаются они только цветом заливки: один красный, другой синий. Все общие свойства нужно описать в классе элемента или блока (в зависимости от ситуации), а различия в цвете в классах модификаторов.